### ISSUE TITLE
support of txt-based formats

### ISSUE THREAD

**Issue by ghost** [schismtracker/schismtracker#103]
Would be great if schismtracker could support txt-based formats.

A Vortex-Tracker format example can be seen at http://pastebin.com/raw/hYVm4AdY 

A txt version of .mod format would be something like https://pastebin.com/raw/LxPCZkgu (txt version of formats like .s3m, .xm, .it, .mptm seems perfectly simple and possible as well)

What do you all think about?

**Comment by jangler** [comment]
I don't see the point in supporting text versions of MOD, S3M, XM, and IT, since Schism already supports importing all those file types. I can see why text-based equivalents of these formats could be useful (in fact, I wrote an IT-to-text/text-to-IT converter a while back but lost the code) but IMO that conversion is something that's best handled by an outside software, since there's no such standard for this sort of thing anyway.

I'm aware of the VT2 format but I think Schism should only concern itself with import and export of sample-based module files.

I know FamiTracker does have text export (and maybe import?) just for its own use, I believe contributed by rainwarrior/bradsmith.

In summary I don't think this is within Schism's scope. If someone else believes otherwise, feel free to comment and/or reopen the issue if you have that capability.
